static/piped-instances.json

```diff
@@ -0,0 +1 @@
+["https://pipedapi.kavin.rocks","https://pipedapi.tokhmi.xyz","https://api-piped.mha.fi","https://watchapi.whatever.social","https://piped-api.garudalinux.org","https://pipedapi.leptons.xyz","https://piped-api.lunar.icu","https://ytapi.dc09.ru","https://pipedapi-libre.kavin.rocks","https://pa.mint.lgbt","https://pa.il.ax","https://piped-api.privacy.com.de","https://pipedapi.esmailelbob.xyz","https://api.piped.projectsegfau.lt","https://pipedapi.in.projectsegfau.lt","https://api.piped.privacydev.net","https://pipedapi.palveluntarjoaja.eu","https://pipedapi.smnz.de","https://pipedapi.adminforge.de","https://pipedapi.qdi.fi","https://piped-api.hostux.net","https://pipedapi.chauvet.pro","https://pipedapi.jotoma.de","https://pipedapi.pfcd.me","https://pipedapi.frontendfriendly.xyz"]
```

Comment:
Being a hypocrite for commenting this because i hate how this is formatted. Can we make it more readable like https://github.com/FreeTubeApp/FreeTube/blob/development/static/invidious-instances.json?

Reply:
As it's auto-generated by `yarn run refetch-piped-instances`, there shouldn't really be a reason for us to manually edit it anyway.

Reply:
The way i set this up is that we generate this file from a script so we dont have to manually edit this file at all. I can try to make the script format it better if you still think this is still an issue.

Reply:
Oh if manual editing is never needed then i dont care about how it is formatted

Reply:
It should be fairly straight forward to get the script to work for invidious instances as well.